reset() documentation issue (#783)
scikit-hep/boost-histogram: README.md

  * `.values(flow=False)`: Get a view on the values (counts or means, depending on storage)
  * `.variances(flow=False)`: Get the variances if available
  * `.counts(flow=False)`: Get the effective counts for all storage types

>   * `.reset() and .clear()`: Set counters to 0 (Note: It doesn't clear growing axis)

Reviewer: ```suggestion
  * `.reset()`: Set counters to 0 (growing axis remain the same size)
```
